examples/platforms/da15000/radio.c

```diff
@@ -84,7 +84,11 @@ static uint32_t sleep_const_delay;
 
 static otInstance *First_Instace;
 
-#define RF_MODE_BLE             0
+uint32_t NODE_ID = 1;
+
+#define RF_MODE_BLE                (0)
```

Comment:
Should these have a prefix?  like DA15000_RF_MODE_BLE? I'm not sure, and I cant find a good example in the other platforms.